src/metabase/driver/postgres.clj

```diff
@@ -766,3 +766,8 @@
     ::upload/boolean     "BOOLEAN"
     ::upload/date        "DATE"
     ::upload/datetime    "TIMESTAMP"))
+
+(defmethod driver/table-name-length-limit :postgres
+  [_driver]
+  ;; https://www.postgresql.org/docs/current/sql-syntax-lexical.html#SQL-SYNTAX-IDENTIFIERS
```

Comment:
per document this is not a constant but configurable via `NAMEDATALEN`.

should we take the extra miles and query it from DB instead?

otherwise, I think we should have a note in this implementation.